llvm/lib/Target/PowerPC/PPCISelLowering.cpp

```diff
@@ -198,6 +198,9 @@ PPCTargetLowering::PPCTargetLowering(const PPCTargetMachine &TM,
     }
   }
 
+  if (!Subtarget.hasP10Vector())
```

Comment:
How is hasP10Vector() related to the instructions in question?  Do we have testcases for what's generated when hasP10Vector() is enabled?